### PR TITLE
readme

### PR DESCRIPTION
**Pull request by web-dave**
I listed the npm cmd availabe in this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,20 +27,20 @@ $ git clone --depth=1 https://github.com/juristr/webpack-typescript-starter.git
 ```
 Maybe remove the `.git` directory and start with a fresh one.
 
-run `npm i` to install all dependencies.
+run `$ npm i` to install all dependencies.
```
Comment (juristr) [inline README.md]: I'd remove the `$` here. Usually if I mention commands inline I don't add it as prefix. Rather I do it when having multi-line code which represents terminal output like

```
$ echo Hi
Hi
```

:smile:

File: README.md [hunk 2]
```diff
@@ -27,20 +27,20 @@ $ git clone --depth=1 https://github.com/juristr/webpack-typescript-starter.git
...
 
 ### available commands (npm run xxx):
```
Comment (juristr) [inline README.md]: Or better even, maybe we could structure this part as they do it on the AngularClass webpack starter: https://github.com/AngularClass/angular2-webpack-starter#quick-start

What do you think? I quite like it to be honest

Comment (web-dave) [reply README.md]: Yes it looks very nice.

Oh! and I missed to 'cd' into the folder.

**Comment by web-dave** [comment]
I pushed it wellfomed ;)

**Comment by juristr** [comment]
:heavy_check_mark: thx :smile:

**Pull request merged**